content/2015.12.what-can-kinto-do.rst

```diff
@@ -0,0 +1,130 @@
+=========================
+What can Kinto do for me?
+=========================
+
+:date: 2015-12-26
+:slug: what-can-kinto-do-for-you
+:lang: en
+:url: what-can-kinto-do-for-you
+:summary: Kinto, the project we are working on, can do a bunch of different
+          things, but it's sometimes hard to see what it is really good at,
+          and where it would make sense to use other solutions.
+
+Kinto — the project we are working on since quite some time now — can do a bunch
+of different things for you and be useful in many different situations.
+
+However we found out that it is really hard to understand at a glance what it
+can do, and where it could help you build your projects in a better way.
+
+Here is an attempt to fix this.
+
+
+Kinto is not a framework
+########################
+
+First and foremost, Kinto is not a framework. Kinto is meant to replace the
+storage layer of your application, or more accurately, the synchronisation
+abilities of your applications.
+
+Said differentely, you store your data locally for your application and Kinto
+will make this data available to different devices. That means that you can use
+Kinto with React, Angular, JQuery (in the JavaScript world) or any framework in
+any language. Kinto abstracts the synchronisation of the application's data.
+
+Some examples
+#############
+
+Let's take some examples. A few weeks ago we prepared a bunch of demos
+showcasing what Kinto can do and how easy it is. In order to do that, we
+decided to take a few applications we liked that are usually not doing
+synchronisation and sharing. There are many out there, and we chose just a few of
+them:
+
+- **A JQuery calendar application** which let you add new events and see them
+  on a calendar.
+- **A leaflet map** where you can add new markers and move them easily.
+- **A TODO-list application**, because we actually really like TODO lists.
+
+And then we decided to make them collaborative apps, using Kinto. The goal here
```

Comment:
'collaborative' sort of implies that one data set is shared by multiple people - is that what you mean?
You could also refer to 'task continuation': start on one device, and continue editing the same data on another.